src/is-android.ts

```diff
@@ -0,0 +1,5 @@
+export default function isAndroid(ua): boolean {
```

Comment:
I'm a little bit surprised (and quite pleased) that the ts-eslint rules require explicit return types for booleans.